.ci/Jenkinsfile

```diff
@@ -202,7 +172,8 @@ pipeline {
                       sshagent(['f6c7695a-671e-4f4f-a331-acdce44ff9ba']) {
                         sh '.ci/prepare-git-context.sh'
                         sh 'gem install rake yard rspec'
-                        sh 'rake release'
+                        sh 'touch file'
```

Comment:
Disabling it for now?

Reply:
I'll revert https://github.com/elastic/apm-agent-ruby/pull/549/commits/20ba9c1ab94f3a3226fb8db1cac8843856ebc152 

I just wanted to validate every single stage does behave as expected and no weird chmod or similar privileges for certains files could affect the following stages